_component_design/modal.md

```diff
@@ -46,23 +46,41 @@ helix-ui-javascript: false
 
 <section class="static-section" markdown="1">
 
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
 ## Specifications
+Use the following specifications when you construct modals.
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
 
 <div class="hxRow"  markdown="1">
 
 {% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
 
-### Spacing
+### Composition
+A modal contains the following:
```

Comment:
Also, s/`the following`/`the following elements` (see the last row of the table at http://rackerlabs.github.io/docs-rackspace/style-guide/style/lists.html?highlight=following+adjective#introductory-text)